data/data.go

```diff
@@ -65,3 +65,71 @@ type StringModifiers struct {
 	I        bool `json:"i"` // for regex
 	S        bool `json:"s"` // for regex
 }
+
+type Keyword struct {
+	Name string
+}
+
+type Expression struct {
+	Left        interface{}
+	Operator    string
+	Right       interface{}
+}
+
+// #Identifier
+type StringCount struct {
+	Identifier string
```

Comment:
For `StringCount`, `StringOffset`, `StringLength`, and any other expression dealing with string identifiers, I would like them to contain a pointer to the `String` itself.  I submitted a PR for something that could help with this: https://github.com/CapacitorSet/yara-parser/pull/1

What would be really great is if we could do something like:

```go
s1 := data.String {
    ...
}

rule.Condition = data.Expression{s1}
```

I'm not sure how to implement something like that at this point, yet, though.